### ISSUE TITLE
functions doesn't work inside dynamic created property

### ISSUE THREAD

**Issue by SkyKnight** [WorkMaze/JUST.net#193]
Hi,

my input json:

`{
        "sections": [
            {
            "id": "first",
            "label": "First section"
            },
            {
            "id": "second",
            "label": "Second section"
            }
        ]
    }`

transformer json:

`{
        "areas":
        {
            "#loop($.sections)":
            {
                "#eval(#currentvalueatpath($.id))": {
                    "description": "#currentvalueatpath($.label)"
                }
            }
        }
    }`

result:

`{
    "areas": [
        {
            "first": {
                "description": "#currentvalueatpath($.label)"
            }
        },
        {
            "second": {
                "description": "#currentvalueatpath($.label)"
            }
        }
    ]
}`

expected result:

`{
    "areas": [
        {
            "first": {
                "description": "First section"
            }
        },
        {
            "second": {
                "description": "Second section"
            }
        }
    ]
}`

Looks like bug for me, but maybe I missed something on readme :)

**Comment by Courela** [comment]
Yes, that's a bug, it only parses the value expression in #eval if it is a string, being an object it just outputs what's in the transformer.
